uint8[4] id
int32 latitude
int32 longitude
uint16 elevation
uint8 transmission
uint16 speed
uint16 heading
int16 angle
int16 acc_long
int16 acc_lat
uint8 acc_vert
int16 acc_yaw
uint8 wheelbrakes
uint8 traction
uint8 abs
uint8 scs
uint8 brakeboost
uint8 auxbrakes
st_CFM[] covoyfoamtion
st_CSM[] covoystatus
st_CLCM[] convoylanechange
